api/proto/openapiv1/dop/dop.proto

```diff
@@ -2905,10 +2906,10 @@ service dop {
     };
     rpc CMDB_ITERATION_DELETE (CMDB_ITERATION_DELETE_Request) returns (google.protobuf.Empty) {
         option (google.api.http) = {
-            delete: "/api/iterations/{id}",
+            delete: "/api/iterations/{id}/{onlyIteration}",
```

Comment:
this change will let original proxy invalid.

Reply:
ok，modified